pkg/apis/eksctl.io/v1alpha5/types.go

```diff
@@ -265,6 +267,8 @@ const (
 	NodeGroupTypeManaged NodeGroupType = "managed"
 	// NodeGroupTypeUnmanaged defines an unmanaged nodegroup
 	NodeGroupTypeUnmanaged NodeGroupType = "unmanaged"
+	// NodeGroupTypeUnowned defines an unmanaged nodegroup
```

Comment:
Is this comment correct?

Reply:
oops! Bad copy pasta from above. good spot